c_glib/arrow-glib/reader.hpp

```diff
@@ -21,6 +21,7 @@
 
 #include <arrow/api.h>
 #include <arrow/csv/api.h>
+#include <arrow/json/api.h>
```

Comment:
Could you put this after `arrow/ipc/...`?
We list `#include` in alphabetically order.

Reply:
I've fixed it.